# Velocity setpoint in NED frame
float32 vx		# in m/s NED
float32 vy		# in m/s NED
float32 vz		# in m/s NED
